File: server/scripts/generateUsers.js
const { faker } = require("@faker-js/faker");

const generateUser = () => ({
  email: faker.internet.email(),
  password: "password",
  username: faker.internet.username(),
  createdAt: new Date().toISOString(),
});
const addUser = (number = 0) => {
  const users = [];
  for (let i = 0; i < number; i++) {
    const user = generateUser();
    users.push(user);
  }
  return users;
};

module.exports = {
  addUser,
};

File: server2/scripts/generate_users.py
from faker import Faker
from datetime import datetime

fake = Faker()


def generate_user():
    return {
        "email": fake.email(),
        "password": "password",
        "username": fake.user_name(),
        "createdAt": datetime.utcnow().isoformat()
    }


def add_user(number=0):
    users = []

    for _ in range(number):
        users.append(generate_user())

    return users